app/gui/view/debug_scene/component-list-panel-view/src/lib.rs

```diff
@@ -209,6 +208,12 @@ pub fn main() {
             size <- all_with(&init, &panel.size, |(), panel_size| *panel_size);
             snap <- all_with(&size, &scene.frp.shape, |sz, sh| snap_to_pixel_offset(*sz, sh));
             eval snap((snap) panel.set_xy(*snap));
+
+
+            // === Disable navigator on hover ===
+
+            navigator.frp.disable <+ panel.is_hovered.on_true();
+            navigator.frp.enable <+ panel.is_hovered.on_false();
```

Comment:
If you can, try doing 1 FRP input, like `navigator.set_enabled` . This makes other FRP places easier to write.